docker/install-scrypted-dependencies-win.ps1

```diff
@@ -44,30 +52,19 @@ var svc = new Service({
     },
   ]
 });
-
-svc.on('alreadyuninstalled', () => {
-  svc.install();
-});
-
-svc.on('uninstall', () => {
-  svc.install();
-});
-
-svc.on('error', () => {
-  svc.install();
-});
-
 svc.on('install', () => {
-  svc.start();
+  console.log("Service installed");
 });
-
-svc.uninstall();
+svc.install();
 "@
 
 $INSTALL_SERVICE_JS_PATH = $SCRYPTED_HOME + '\install-service.js'
 $INSTALL_SERVICE_JS | Out-File -Encoding ASCII -FilePath $INSTALL_SERVICE_JS_PATH
 
 node $INSTALL_SERVICE_JS_PATH
 
+# Manually start service, node-windows has issues starting service
+sc start scrypted.exe
```

Comment:
The `svc.start()` script didn't work for me. Even though `node-windows` reports is started successfully, I found that the service showed as "stopped" in the service application. I find `sc start` work more reliably.

Reply:
I ended up removing this since I fixed `node-windows` itself